include/matx/operators/scalar_ops.h

```diff
@@ -170,8 +170,38 @@ template <typename T> struct SqrtF {
   }
 };
 
-template <typename T> using SqrtOp = UnOp<T, SqrtF<T>>;
+template <typename T>
+static __MATX_INLINE__ __MATX_HOST__ __MATX_DEVICE__ auto _internal_csqrt(T v1)
+{
+  if constexpr (is_cuda_complex_v<T>) 
+  {
+    return cuda::std::sqrt(v1);
+  }
+  else {
+    ///\todo TYLER_TODO should probably protect for only float types
+    return sqrt(static_cast<cuda::std::complex<T>>(v1));
+  }
+  if constexpr (!is_cuda_complex_v<T>) {
```

Comment:
This shouldn't be needed anymore. I think we're going to clean these up and dump the old 11.4 requirement

Reply:
I believe this is handled with the redesign above. let me know if I'm missing any other condition you want to cover.